### PR TITLE
add site

### PR DESCRIPTION
**Pull request by MCFlatliner**

### CODE REVIEW

File: all.json [hunk 1]
```diff
@@ -32,5 +32,7 @@
     "polkawallets.site",
     "polkodot.network",
     "walletsynchronization.com"
+    "polkadot-distribution.live"
+    "buncebit.com"
```
Comment (jacogr) [inline all.json:36]: Not sure about this one?

Comment (MCFlatliner) [reply all.json:36]: you are asking me, or you are not sure yourself?

File: all.json [hunk 2]
```diff
@@ -32,5 +32,8 @@
     "polkawallets.site",
     "polkodot.network",
     "walletsynchronization.com"
+    "polkadot-distribution.live"
```
Comment (jacogr) [inline all.json:35]:
```suggestion
    "polkadot-distribution.live",
```

Also needs a `,` at the end of the previously line.

File: all.json (hunk 1 above)
Comment (jacogr) [reply all.json:36]: Yes, not sure, I looked at it and it looks like a exchange - also cannot find it on any other crypto scam list.

File: all.json (hunk 2 above)
Comment (MCFlatliner) [reply all.json]: new to this, have to learn the details. i changed it and updated

File: all.json (hunk 1 above)
Comment (MCFlatliner) [reply all.json]: it was reported in scam section at polkadot (official)

Comment (jacogr) [reply all.json]: Here is a reference - https://twitter.com/dubstard/status/1371863286426312705?s=20

so indeed

File: all.json (hunk 2 above)
Comment (jacogr) [reply all.json]: No issues at all.

File: all.json [hunk 3]
```diff
@@ -55,7 +56,9 @@
     "polkadotwallet-unlock.org",
     "polkawallets.site",
     "polkdot-live.network",
+    "polkadot-distribution.live",
     "polkodot.network",
+    "wallet-syncing.com",,
```
Comment (jacogr) [inline all.json:61]:
```suggestion
    "wallet-syncing.com",
```

**Review by jacogr — APPROVED** [review]

**Comment by jacogr** [comment]
Thank you!

**Pull request merged**